source/WindowScroller/WindowScroller.js

```diff
@@ -127,6 +131,8 @@ export default class WindowScroller extends PureComponent {
 
   // Referenced by utils/onScroll
   __handleWindowScrollEvent (event) {
+    if (!this._isMounted) return
```

Comment:
I'm willing to accept this fix, but I'd like to know more about the case that it's fixing so I can determine if there's a better fix. Can you provide a Plnkr (or similar) repro @liorbrauer ?